frontend/public/components/routes/create-route.tsx

```diff
@@ -637,7 +637,7 @@ const AlternateServicesGroup: React.FC<AlternateServiceEntryGroupProps> = (props
 
   React.useEffect(() => {
     props.onChange({ name, weight }, props.index);
-  }, [name, weight]);
+  }, [name, props, weight]);
```

Comment:
Sorry for not catching this earlier.  Please omit this linter fix since it's unrelated.

Reply:
removed